docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,628 @@
+
+The page provides a high-level overview, integration steps, and links to additional documentation.
+
+UID2 provides a mobile SDK for Android and iOS with the following features:
+
+- UID2 token generation
+- Automatic refreshing of UID2 tokens
+- Automatic storage of UID2 tokens
+
+To integrate with UID2 client-side, you will need to complete the following steps:
+
+1. Complete UID2 account setup.
+
+1. Add a UID2 mobile SDK into your mobile application.
+
+1. Configure the UID2 Mobile SDK.
+
+1. Check that the token was successfully generated and then pass it for bid stream use.
+
+1. Optionally, integrate the UID2 GMA/IMA Plugin for GAM Secure Signal integration.
+
+
+It includes the following sections:
+
+- [UID2 Mobile SDK Version](#uid2-mobile-sdk-version)
+- [Client-Side Integration Example](#client-side-integration-example)
+- [Complete UID2 Account Setup ](#complete-uid2-account-setup)
+- [Add the UID2 Mobile SDK to Your Mobile Application](#add-the-uid2-mobile-sdk-to-your-mobile-application)
+- [Configure the UID2 Mobile SDK](#configure-the-uid2-mobile-sdk)
+- [Format Examples for DII](#format-examples-for-dii)
+- [Token Storage and Refresh](#token-storage-and-refresh)
+- [Check that the token was successfully generated and then pass it for bid stream use](#check-that-the-token-was-successfully-generated-and-then-pass-it-for-bid-stream-use)
+- [Best Practice on When to Pass DII to the UID2 Mobile SDK](#best-practice-on-when-to-pass-dii-to-the-uid2-mobile-sdk)
+- [Opt-Out Handling](#opt-out-handling)
+- [Enable Logging](#enable-logging)
+- [Optional: UID2 GMA/IMA Plugin for GAM Secure Signal integration](#optional-uid2-gmaima-plugin-for-gam-secure-signal-integration)
+
+## UID2 Mobile SDK Version
+
+To follow this client-side integration guide, it requires v1.0.0 or higher of UID2 SDK for Android and iOS. Follow the Add UID2 Mobile SDK into your mobile application section below to install these SDK versions in your applications.
+
+## Client-Side Integration Example
+
+For an example of how to configure a UID2 mobile SDK, and how to generate tokens using client-side integration for mobile, you can try out the UID2 development application.
+
+#### For Android:
+
+1. Check out the main branch of the [UID2 SDK for Android source code repository on GitHub](https://github.com/IABTechLab/uid2-android-sdk/tree/main).
+1. Open the checked out directory in Android Studio.
+1. Run the **dev-app** application.
+
+#### For iOS:
+
+1. Check out the [main branch of the UID2 SDK For iOS source code repository on GitHub](https://github.com/IABTechLab/uid2-ios-sdk/tree/main).
+1. In Xcode, open this project file:
+
+   ```
+   Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp.xcodeproj
+   ```
+1. Run the **UID2SDKDevelopmentApp** app scheme.
+
+(**GWH__SW I'm not sure if this whole section should be the procedure, rather than just steps 1 and 2?**)
```

Comment:
i think it's latter if i understand you correctly but can confirm when we next zoom

Reply:
Resolved in meeting, thx.